packages/gestalt/src/Icon.js

```diff
@@ -28,6 +28,7 @@ type IconProps = {
     | 'white',
   inline?: boolean,
   size?: number | string,
+  title?: string,
```

Comment:
If we are adding this as a prop to `Icon` it will be exposed to users, can we update the docs and the react prop types as well?

Reply:
Sure, can do.